### DISCUSSION TITLE
【代理人交流】斐济杯使用体验报告

### DISCUSSION THREAD

**Discussion by Manbow1997** [share121/inter-knot#2822] [其他]
![2adb7cab9b10ddacb6461d215f372f08](https://github.com/user-attachments/assets/3b81b9aa-e7a6-4658-a478-f5ce4a7118b8)

### **不是广告！不是广告！不是广告！**

**一.前言**

**1.**
子曰：食色性也。弗洛伊德，在其性学说中，也指出：性欲极其能量生来即有。饿了就要吃东西，有需求就要解决，这是人天生的、自然的生理需求，理智看待，谈性色变或沉溺情色都太过，中庸辩证的看待才是正解。

**2.**
对于需求，憋着难受，“自导自演”又伤牛牛，还不想出去“吃外卖”，只得使用“虚拟女友”、“赛博伴侣”，尝以饱腹罢了。不过还需提醒各位：“自娱自乐”虽好，时机恰当，还要交往朋友，回归现实。戒色禁欲，或是频繁放纵，都对身体不好，节制有度、规划控制，方为上法。个人认为频率最好在一周2~3次。

**3.**
目前，就个人感受，在【新艾利都】内部环境中，谈性色变，关于这方面知识，引导教育多是闭口不谈，杯子这些东西，更是上不得台面，且极易社死。理论知识匮乏，又何谈战斗实操？。理论知识、实践活动，全靠自觉自悟。

**4.**
本人于2024年8月13下单了两款斐济杯，并于17日使用了其中一款，杯子也很满意。此贴主要在分享个人感受和体验，要是对各位有些许成长帮助，就算成功！不知道在绳网能不能说？

**5.**
如果我的帖子影响不好或者就不想讨论杯子又或者嘲笑笔者云云，我会删除，但千万别骂我，本人有心理疾病，你不开心，你骂我，我也难过。

**二.正文**

**1.杯子**
**①基本信息**
基本信息是看网页介绍的，掺杂一点个人感受。
品牌名称：LOVEFACTOR。
产品名称：禁忌XX。
产品重量：约930G。2斤差点，没概念的可以找旁边东西掂量掂量，个人认为不太重。
腔长：13.5cm。
全长：约18.5cm。
产品材质:TPE。
产品类型：非贯通。贯通的话，虽清洗方便，不过体验大打折扣，个人还是偏向非贯通的。
**②特点**
**（1）入口空腔结构，紧致突破；**
**（2）悬吸式圣所禁地，发射流畅；**
**（3）中高刺激，新手慎入。**

第一眼看上去是有点偏大，两手持握差不多。
出油量有点大，就拿出来摸了下，手就油油的。不过本人不太在乎这个，“不管干杯不干杯，能嘬NiuNiu的就是好杯”。

**2.使用体验**
**①.入口**
下部，外观是馒头型的，椭圆状的包裹在外面，手指摸索，能感觉到一个空腔，两手打开，也确实如此。此时，打开看到的，也就是真正的入口。开头还是有点紧的。第一次使用不知道，进出时候滑出来了，怼到空腔处，就是进不去，还顶起个泡，拿起细细端详，才发现：还有这洞天设计！
举个例子，像朵含苞待放的花。一个花骨朵，外面看，娇艳欲滴；待你从中，把花瓣朵朵扒开，就看到花心了。
这就带来了两种体验：
**（1）进入的时候。**
有时候会怼歪了，怼到空腔处。当然这种情况不太常见，一是一直在入口摩擦，会对不准；二是冲刺幅度过大，会对歪开来；
**（2）出来的时候。**
外圈隔绝了空气，进入时候空气都被排空了，这就导致，进的越深，吸力越大，待往外退的时候，杯子就会嘬着你的哞哞，咬着你叫你撤退艰难。BullBull 皮肤被嘬着，但头被包在里面，就像剥香肠皮，皮被扯得有点痛痛的。不过解决办法就是，出入的时候，双手捏着往两边张一点，放点空气到空腔处，就好了。

这样的话，如果你想尝试高压作业、排气施法，正常把玩就行；如果你觉得太紧了有点受不了，就照解决办法做就好了。

**②内部**
杯子整体体验，偏中上刺激：入口紧致，通道曲幽深邃，紧致肉浪，连环拍打，最终到达悬吸禁地。刺激度较高的在中段，出入的时候能感觉到，触电般，阵阵酥麻。这种感受，就像盗墓贼被早已准备好的墓穴机关轮番伺候，最后还大难不死盗得宝藏，但也汗流浃背。本人使用完，也得休息两三天，太能“伺候”了。

**③顶部**
顶部的话，看介绍，有个2.5cm的“小仓库”，进入的时候，“初极狭，才通人。复行数十步，豁然开朗“；或者像林克从神庙醒来，走出洞穴，站在山崖上望去，一片盎然碧绿（不过过程相反，体验相同）。BullBull 在里面还是挺舒服的，左转右钻，上蹿下跳，像西班牙斗牛一样，顶人顶红眼了。这时可以用手，隔着杯子顶部，捏捻揉搓，调整弹道，降低阈值。发射升空，头部也不会感到明显堵塞，等到真正倒计时时......3—2—1—摧枯拉朽、畅通无阻，就像......就像古德里安带领他的19装甲军，后接庞大坦克军群，狂飙法兰西，直冲敦刻尔克。

**3.个人评价**
满分10分的话，个人给10分。外表偏大，肉肉的，本人也喜欢吃馒头，很对胃口。还有，本人也喜爱在高压环境作业。最后，杯子入口结构，是用完后，端详研究之下，才发现的，算是意外之喜！这下更喜欢了。

**三.**
本帖应该不算涩情内容吧，不知道在绳网能不能发出来！
另一款斐济杯，也挺对本人XP的，黑皮辣妹。
还有有关杯子使用后，如何清洗，如何吸水除湿，还有如何保养这些，有时间再写吧！

**Comment by TetsuyaTetsuya** [comment]
好好好，绳匠在写500丁尼的，孩子爱看(๑♡3♡๑)

**Reply by Manbow1997** [reply]
> 好好好，绳匠在写500丁尼的，孩子爱看(๑♡3♡๑)

乔普师傅的拉面，一碗都5000（傲娇.JPG

**Comment by xiaolicidao** [comment]
这种帖子居然真的能发出来
![8c1540c8-2d60-4611-86c9-277d1b305d23](https://github.com/user-attachments/assets/6e580782-b105-4214-89b0-01fab52a90d5)

**Reply by start63** [reply]
逆天，派蒙杯子

**Reply by xiaolicidao** [reply]
太逆天了...

**Reply by Manbow1997** [reply]
> 逆天，派蒙杯子
只是搜过，没有派蒙的。个人感觉派蒙挺可爱的还小小的！

**Reply by 1luik** [reply]
国外比较开放，github是国外，基本上全世界都在用的

**Reply by opqphp** [reply]
> > 逆天，派蒙杯子
> 只是搜过，没有派蒙的。个人感觉派蒙挺可爱的还小小的！
> 

![Screenshot_2024-08-19-19-21-03-59_40d30947b37419455897df6aa3094b56.jpg](https://github.com/user-attachments/assets/54c4c580-9556-4ffb-bab2-e7080bb3ac68)

**Comment by Leo2126** [comment]
逆大天

**Comment by nostalgiatan** [comment]
虽然逆天，但确实没有违反一条规定。
这种内容也是第1次见，无敌了。

**Comment by Manbow1997** [comment]
如果影响不好，文章将删除，对不起，我不该写这么逆天的东西。

**Reply by nostalgiatan** [reply]
不，无所谓。
这个社区本来就没有任何范围，只要不突破界定的规则。
影响？能有啥影响？本来我们这个网站也不是12+的，只要没有违背规则发啥都无所谓。

**Reply by VacuolePaoo** [reply]
请对文章封面的敏感内容进行打码或修改处理
正文描述请做出适当修改
> 绳网是一个16+社区，发点大家能看的

**Reply by Manbow1997** [reply]
> 请对文章封面的敏感内容进行打码或修改处理 正文描述请做出适当修改
> 
> > 绳网是一个16+社区，发点大家能看的

改好了。看下哪里还需要修改？

**Comment by aizai1173** [comment]
应该没事，能进绳网的小孩不会有太多，大概...

**Reply by LuAn-afl** [reply]
小学生不存在吧光一个注册和魔法就难住了99%的小学生我一个高三的差点都没进来

**Reply by YuTinMIn** [reply]
让你失望了，我六年级刚毕业

**Comment by Kirolei** [comment]
这个试用委托我接了！（bushi）

**Comment by CJHXAMSTLP** [comment]
寄用用我

**Comment by samo114514** [comment]
给我用用
![Uploading 77fc2df634733d979bbb22ea7e3ffa6.jpg…]()

**Comment by NOVAQUQ** [comment]
![IMG_20240818_235021](https://github.com/user-attachments/assets/636629c9-b503-4f3b-8fa8-a7fc31499cd5)
![img_-455464463_1713013017255](https://github.com/user-attachments/assets/22f489c2-230c-4eff-96e6-db777068c8f4)

**Reply by Manbow1997** [reply]
> ![IMG_20240818_235021](https://private-user-images.githubusercontent.com/177938786/358918399-636629c9-b503-4f3b-8fa8-a7fc31499cd5.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwNjAwNzEsIm5iZiI6MTcyNDA1OTc3MSwicGF0aCI6Ii8xNzc5Mzg3ODYvMzU4OTE4Mzk5LTYzNjYyOWM5LWI1MDMtNGYzYi04ZmE4LWE3ZmMzMTQ5OWNkNS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxOVQwOTI5MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00ZTlhODc2ZDg4YzM1YjVjYjYwYjY4OTg5MWVmZTNlNDdjYmFiNWQ3ZmEyYzVhNjBmMGJjZjMyNjBiNjJkOTk3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.7hT4vLxFe-r7X5uY8JPzRS8G-47Kh0zEXTyw1sCORec) ![img_-455464463_1713013017255](https://private-user-images.githubusercontent.com/177938786/358918435-22f489c2-230c-4eff-96e6-db777068c8f4.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwNjAwNzEsIm5iZiI6MTcyNDA1OTc3MSwicGF0aCI6Ii8xNzc5Mzg3ODYvMzU4OTE4NDM1LTIyZjQ4OWMyLTIzMGMtNGVmZi05NmU2LWRiNzc3MDY4YzhmNC5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MTlUMDkyOTMxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzMwOGE3NWUxODY2MmMyMzMyNmY2ZDZjNjczOGM1MjZlYzJlNDhhNDUwOTVhMzYxYzA0ZTE5MmIyY2UyNDk1OCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.wAkk7-ec7dZsjho_ktX79-xUvqKJwhUc_35ozSy1iXQ)

能不能把第二张AI图删了，每次看评论区都要翻一会（
![0073686F](https://github.com/user-attachments/assets/e9196411-652f-4bd0-9866-177c9c0e4d19)

**Comment by Kumorin-chika** [comment]
你也逆天

**Comment by wer-456** [comment]
逆天！！！！

**Comment by LilangAlan** [comment]
逆天<img src="https://bbs-static.miyoushe.com/static/2023/11/07/5f0581151a00ca4efe963928c2607df0_5889753033090282341.png" alt="尾巴 就这" width="75" >

**Comment by LIDOUOVO** [comment]
逆天

**Comment by bangboo12123** [comment]
逆天
![Uploading 23.jpg…]()

**Comment by meillml** [comment]
逆天

**Comment by VacuolePaoo** [comment]
十八手斐济杯

**Reply by Manbow1997** [reply]
> 十八手斐济杯

降牛十八手？

**Comment by Tartaglia770** [comment]
我才14岁啊啊啊

**Reply by Manbow1997** [reply]
> 我才14岁啊啊啊

开头已提醒：如果你还未成年，请你离开，这是大人才能看的！

**Comment by Legendary-agent** [comment]
【反馈】不好用，别信，你们可以试试我。

**Comment by Zhangshibo-sudo** [comment]
这绳网没有审核吧，只有作者。除了极端的，估计都能发

**Reply by VacuolePaoo** [reply]
哦？

**Comment by wall-leg-sao-gang** [comment]
这就是16+游戏吗

**Comment by sz0628** [comment]
修女是吗，我还是喜欢用御神子（doge）

**Reply by Manbow1997** [reply]
> 修女是吗，我还是喜欢用御神子（doge）

御神子一代纯慢玩，适合新手，低刺激；二代中刺激，可进阶。修女是高刺激款的！

**Comment by lcx19527** [comment]
逆天

**Comment by 1492830678** [comment]
太逆天了哥们

**Comment by LMYMH** [comment]
逆天！

**Comment by hanshui2077** [comment]
这个委托我劫了

**Comment by 660W** [comment]
逆天，不过我喜欢

**Comment by ybbgh** [comment]
❤点❤我❤观❤看❤邦❤布❤可❤爱❤录❤像❤

**Comment by Luma-zzz** [comment]
逆了

**Comment by Guessme404** [comment]
实验时长不够建议增加

**Comment by guanyq000** [comment]
好活，打赏50丁尼

**Comment by Josiku** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/11/07/feb46dc482bc621cc68058b0f77478d6_3301492614879353942.png" alt="桂乃芬 惊" width="75" >

**Comment by Featherot** [comment]
太6了

**Comment by azlim123** [comment]
逆大天啊哥们

**Comment by poi0v0** [comment]
可惜现在不能关注收藏

**Comment by CRZZA** [comment]
逆天

**Comment by tohka-722** [comment]
一周三次真的不会虚吗

**Reply by Manbow1997** [reply]
> 一周三次真的不会虚吗

一周三次都觉得多，是不是该锻炼一下？比如做做深蹲什么的

**Comment by aboluo12138** [comment]
哇，怎么还有杯子评测的啊

**Comment by opqphp** [comment]
![Screenshot_2024-08-19-18-57-11-06_40d30947b37419455897df6aa3094b56](https://github.com/user-attachments/assets/13f6b198-9c1b-4fb9-bd81-c2c31bbd1376)

**Comment by SakuraSora39** [comment]
逆天，不过在这里，倒也正常

**Comment by qiuyezhi** [comment]
88节都过了

**Comment by Eufo0305** [comment]
太逆天了。

**Comment by Lyou777** [comment]
这里是绳网，这个是绳斤

**Comment by Timekeeper2006** [comment]
给我干哪来了，这还是国内吗？！
![OIP](https://github.com/user-attachments/assets/bd4239b4-8902-449b-ade3-e2ab94e671bc)

**Comment by EHBE888** [comment]
逆天
